content/md/en/docs/reference/substrate-connect.md

```diff
@@ -0,0 +1,200 @@
+#### **@polkadot/rpc-provider**
+This package contains the `substrate-connect` rpc provider that integrates and uses substrate-connect:
+```bash
+yarn add @polkadot/rpc-provider
+```
+*or*
+```bash
+npm i @polkadot/rpc-provider
+```
+
+#### **@polkadot/api**
+
+This library provides a clean wrapper around all the methods exposed by a Polkadot/Substrate network client. It is used with the outcome of rpc-provider above
+
+```bash
+yarn add @polkadot/api
+```
+*or*
+```bash
+npm i @polkadot/api
+```
+
+Below are some examples that shows how the rpc-provider can be used simply with the already integrated networks (e.g. polkadot, kusama, westend, rococo), or with a custom chain by using its chainspec, or connecting to a parachain."
+
+*Simple usage (Using an integrated, suported chain)*
+```js
+import { ScProvider, WellKnownChain } from '@polkadot/rpc-provider/substrate-connect';
+import { ApiPromise } from '@polkadot/api';
+// Create the provider for a known chain
+const provider = new ScProvider(WellKnownChain.westend2);
+// Stablish the connection (and catch possible errors)
+await provider.connect()
+// Create the PolkadotJS api instance
+const api = await ApiPromise.create({ provider });
+await api.rpc.chain.subscribeNewHeads((lastHeader) => {
+console.log(lastHeader.hash);
+});
+await api.disconnect();
+```
+
+*Simple usage (custom chain/chainspec)*
+```js
+import { ScProvider } from '@polkadot/rpc-provider/substrate-connect';
+import { ApiPromise } from '@polkadot/api';
+import jsonCustomSpec from './jsonCustomSpec.json';
+// Create the provider for the custom chain
+const customSpec = JSON.stringify(jsonCustomSpec);
+const provider = new ScProvider(customSpec);
+// Stablish the connection (and catch possible errors)
+await provider.connect()
+// Create the PolkadotJS api instance
+const api = await ApiPromise.create({ provider });
+await api.rpc.chain.subscribeNewHeads((lastHeader) => {
+console.log(lastHeader.hash);
+});
+await api.disconnect();
+```
+
+*Parachains usage*
+
+```js
+import { ScProvider, WellKnownChain } from '@polkadot/rpc-provider/substrate-connect';
+import { ApiPromise } from '@polkadot/api';
+import jsonParachainSpec from './jsonParachainSpec.json';
+// Create the provider for the relay chain
+const relayProvider = new ScProvider(WellKnownChain.westend2);
+// Create the provider for the parachain. Notice that
+// we must pass the provider of the relay chain as the
+// second argument
+const parachainSpec = JSON.stringify(jsonParachainSpec);
+const provider = new ScProvider(parachainSpec, relayProvider);
+// Stablish the connection (and catch possible errors)
+await provider.connect()
+// Create the PolkadotJS api instance
+const api = await ApiPromise.create({ provider });
+await api.rpc.chain.subscribeNewHeads((lastHeader) => {
+console.log(lastHeader.hash);
+});
+await api.disconnect();
+```
+
+## Advanced Usage
```

Comment:
```suggestion
## Add Substrate Connect to other with applications

Reply:
I think - if we want to change this - it should be something like:
```suggestion
## Use Substrate Connect with other libraries/frameworks
```